test/gristWebDriverUtils.ts

```diff
@@ -210,8 +223,11 @@ export class GristWebDriverUtils {
     const max = 10;
 
     // Keep dismissing prompts until there are no more, up to a maximum of 10 times.
-    while (i < max && await this.driver.find('.test-behavioral-prompt').isPresent()) {
-      await this.driver.find('.test-behavioral-prompt-dismiss').click();
+    const getButton = () => {
```

Comment:
My guess is that this change isn't of much benefit, and my preference would be to keep it unchanged, since it's a copy of code from grist-core (which is much more exercised and reliable). For the same reason, `ignoreMissingElementErrors` doesn't seem needed.

Reply:
Rolled this back - totally agree it doesn't make sense making these changes, if we haven't updated gristWebDriverUtils.